docs/policy/create_policy.rst

```diff
@@ -0,0 +1,5 @@
+#Process to publish policy proposals
```

Comment:
This is using markdown syntax, so the file should have a .md extension

Reply:
Ah, got it!